src/AmazonPHP/SellingPartner/Api/AplusContentApi/APlusSDK.php

```diff
@@ -52,6 +52,8 @@ public function __construct(ClientInterface $client, HttpFactory $requestFactory
      *
      * @throws ApiException on non-2xx response
      * @throws InvalidArgumentException
+     *
+     * @return \AmazonPHP\SellingPartner\Model\APlus\PostContentDocumentResponse
```

Comment:
not sure why this was added 🤔  but lets merge it as is and I will debug it later as it has no impact on the code

Reply:
This is after regeneration. Not added manually.

Reply:
I know, probably some rules are overlapping between rector and CS fixer, need to debug it, thanks!